### PR TITLE
Changes URL tests for file path normalisation proposal

### PR DESCRIPTION
**Pull request by alwinb**
Changes to the URL tests (urltestdata.json and setters_tests.json) for the proposal outlined in https://github.com/whatwg/url/issues/405.

### CODE REVIEW

File: url/resources/urltestdata.json [hunk 1]
```diff
@@ -7217,4 +7344,4 @@
     "search": "",
     "username": "user"
   }
-]
+]
```
Comment (annevk) [inline url/resources/urltestdata.json:7347]: Please restore the newline at the end.

**Review by annevk — APPROVED** [review]
Thanks for writing these! This looks correct to me modulo the nit.

File: url/resources/urltestdata.json (hunk 1 above)
Comment (alwinb) [reply url/resources/urltestdata.json]: Ah, I didn't see that.

Comment (alwinb) [reply url/resources/urltestdata.json]: Just so I know how this works, does each such commit send out another approval request to each of the reviewers?

Comment (annevk) [reply url/resources/urltestdata.json]: I think that depends on how you configure your GitHub notifications, but maybe? I usually wait for some kind of comment before taking another look.

**Review by domenic — APPROVED** [review]

**Pull request merged**